examples/src/main/kotlin/examples/visualisation/LeNetFashionMnistVisualisation.kt

```diff
@@ -38,43 +39,50 @@ fun main() {
 
     val (newTrain, validation) = train.split(0.95)
 
-    lenet5().use {
-        it.compile(
+    val sampleIndex = 42
+    val x = test.getX(sampleIndex)
+    val y = test.getY(sampleIndex).toInt()
+
+    lenet5().use { model ->
+
+        model.compile(
             optimizer = Adam(),
             loss = Losses.SOFT_MAX_CROSS_ENTROPY_WITH_LOGITS,
             metric = Metrics.ACCURACY
         )
 
-        it.fit(
+        model.fit(
             trainingDataset = newTrain,
             validationDataset = validation,
             epochs = EPOCHS,
             trainBatchSize = TRAINING_BATCH_SIZE,
             validationBatchSize = TEST_BATCH_SIZE
         )
 
-        val imageId = 0
```

Comment:
is it correct to say, that all these codes will work in Jupyter only?

Also, I suggest keeping all experimental approaches to visualize activations (I agree that it's not so cool as yours) but could you please add your examples separately?

Or what is about creating a separate module like examples or API near and move there a few util functions and lets-plot dependency and adding this module as dependency to examples?

Reply:
No, this code is for jvm as letsplot recognises the plotting engine and use the current available - in this case batik (thats all the beauty ni this solution that we don't have to manually work with windows etc. - just call .show() method on created plots). I run this example trainings on my computer at the end and visualisations were pretty similar to yours samples.

I will add this module (not sure of its name for now) and restore your approach to plotting activations here no to loose it

Reply:
The examples for jupyter need to be done separately in notebooks format if we want them because for know I don't see another option to plot something in jupyter notebook in another way than placing the plot object in the last line of the code cell (I mean calling the .show() method in this case didn't work for me in Datalore)

Reply:
Ok, do you need to separate tickets to add one-two jupyter notebook? Maybe I create later a folder for notebooks and you will add later a few notebooks here.

Reply:
I can add them as a part of this issue as they would be based on your examples which were modified in this issue. I would just need some suggestion where they should be placed